src/Shared/DC/ZRDB/sqltest.py

```diff
@@ -164,6 +164,9 @@ def render(self, md):
                 if not isinstance(v, StringTypes):
                     v = str(v)
                 v = md.getitem('sql_quote__', 0)(v)
+                # fix https://github.com/zopefoundation/Products.ZSQLMethods/issues/14
```

Comment:
The line is too long and is causing the Travis CI checks to fail. Please stay under 80 characters.